// es-abstract/operations/2020.js
'use strict';

module.exports = {
	abs: 'https://262.ecma-international.org/11.0/#eqn-abs',
	'Abstract Equality Comparison': 'https://262.ecma-international.org/11.0/#sec-abstract-equality-comparison',
	'Abstract Relational Comparison': 'https://262.ecma-international.org/11.0/#sec-abstract-relational-comparison',
	AddEntriesFromIterable: 'https://262.ecma-international.org/11.0/#sec-add-entries-from-iterable',
	AddRestrictedFunctionProperties: 'https://262.ecma-international.org/11.0/#sec-addrestrictedfunctionproperties',
	AddWaiter: 'https://262.ecma-international.org/11.0/#sec-addwaiter',
	AdvanceStringIndex: 'https://262.ecma-international.org/11.0/#sec-advancestringindex',
	'agent-order': 'https://262.ecma-international.org/11.0/#sec-agent-order',
	AgentCanSuspend: 'https://262.ecma-international.org/11.0/#sec-agentcansuspend',
	AgentSignifier: 'https://262.ecma-international.org/11.0/#sec-agentsignifier',
	AllocateArrayBuffer: 'https://262.ecma-international.org/11.0/#sec-allocatearraybuffer',
	AllocateSharedArrayBuffer: 'https://262.ecma-international.org/11.0/#sec-allocatesharedarraybuffer',
	AllocateTypedArray: 'https://262.ecma-international.org/11.0/#sec-allocatetypedarray',
	AllocateTypedArrayBuffer: 'https://262.ecma-international.org/11.0/#sec-allocatetypedarraybuffer',
	ArrayCreate: 'https://262.ecma-international.org/11.0/#sec-arraycreate',
	ArraySetLength: 'https://262.ecma-international.org/11.0/#sec-arraysetlength',
	ArraySpeciesCreate: 'https://262.ecma-international.org/11.0/#sec-arrayspeciescreate',
	AsyncFromSyncIteratorContinuation: 'https://262.ecma-international.org/11.0/#sec-asyncfromsynciteratorcontinuation',
	AsyncFunctionStart: 'https://262.ecma-international.org/11.0/#sec-async-functions-abstract-operations-async-function-start',
	AsyncGeneratorEnqueue: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratorenqueue',
	AsyncGeneratorReject: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratorreject',
	AsyncGeneratorResolve: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratorresolve',
	AsyncGeneratorResumeNext: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratorresumenext',
	AsyncGeneratorStart: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratorstart',
	AsyncGeneratorYield: 'https://262.ecma-international.org/11.0/#sec-asyncgeneratoryield',
	AsyncIteratorClose: 'https://262.ecma-international.org/11.0/#sec-asynciteratorclose',
	AtomicLoad: 'https://262.ecma-international.org/11.0/#sec-atomicload',
	AtomicReadModifyWrite: 'https://262.ecma-international.org/11.0/#sec-atomicreadmodifywrite',
	Await: 'https://262.ecma-international.org/11.0/#await',
	BackreferenceMatcher: 'https://262.ecma-international.org/11.0/#sec-backreference-matcher',
	'BigInt::add': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-add',
	'BigInt::bitwiseAND': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-bitwiseAND',
	'BigInt::bitwiseNOT': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-bitwiseNOT',
	'BigInt::bitwiseOR': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-bitwiseOR',
	'BigInt::bitwiseXOR': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-bitwiseXOR',
	'BigInt::divide': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-divide',
	'BigInt::equal': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-equal',
	'BigInt::exponentiate': 'https://262.ecma-international.org/11.0/#sec-numeric-types-bigint-exponentiate',
	'BigInt::leftShift': 'https://262
